### DISCUSSION TITLE
Resize table columns

### DISCUSSION THREAD

**Discussion by sonesay** [chakra-ui/chakra-ui#6681] [Q&A]
Hi, I am using the Chakra Table and need to be able to resize the columns. Out of the box the table does not allow this but is there something we can use on top of the table to allow this behavior?

**Comment by nomadme** [comment]
Yes, this is something that is really needed, and was surprised to find out it was impossible to control the width.

**Comment by joaoh9** [comment]
I am also in need of this feature!

**Comment by hb20007** [comment]
Useful: https://github.com/chakra-ui/chakra-ui/discussions/4449
